### ISSUE TITLE
Request trove classifier

### ISSUE THREAD

**Issue by Cito** [pypa/warehouse#3843]
Request to add a new Trove classifier.

## The name of the classifier you would like to add

* `License:: OSI Approved :: The PostgreSQL License`

## Why do you want to add this classifier?

The PostgreSQL License is used by [PyGreSQL](http://www.pygresql.org/).

Older versions of PyGreSQL used the Python license as Trove classifier, so we were able to upload it to PyPI, but this was actually wrong. PyGreSQL was originally a part of the PostgreSQL source and always said it was licensed according to The PostgreSQL License in its documentation because it considers itself part of the PostgreSQL ecosystem. We want to specify the license now properly on PyPI as well, in accordance with the docs.

The license might be also interesting for other PostgreSQL related software. The PostgreSQL License is in fact OSI approved according to [this list](https://opensource.org/licenses/alphabetical).

This problem prevents us from uploading the actual PyGreSQL version on PyPI because we get this error mesage (at least on TestPyPI):

    HTTPError: 400 Client Error: Invalid value for classifiers. Error: 'License :: OSI Approved :: The PostgreSQL License' is not a valid choice for this field for url: https://test.pypi.org/legacy/

We would be very glad if this could be solved by adding the license as classifier.

**Comment by Cito** [comment]
We cannot upload the latest PyGreSQL release because of this. Can I do anything to help moving this forward?

**Comment by Cito** [comment]
Please give advice how we shall proceed. We currently cannot publish PyGreSQL on PyPI because of this!

**Comment by Cito** [comment]
Thank you. Note that on the official [OSI list](https://opensource.org/licenses/alphabetical) it's listed as "*The* PostgreSQL License"  and "The" is capitalized in the text of the license - this is different from e.g. the "Python License". We might want to follow that nomenclature because it might be intentional and exact wording may be important in legalese.

**Comment by di** [comment]
@Cito On https://www.postgresql.org/about/licence/, it's stylized as "the **PostgreSQL License**" and on https://spdx.org/licenses/ it's just called "PostgreSQL License" so I think we should stick with the existing classifier.

**Comment by Cito** [comment]
You're right - strange that they have it like that on the OSI page then. Thanks again for your help. We will then make that adaption.
